activerecord/lib/active_record/insert_all.rb

```diff
@@ -108,7 +114,6 @@ def unique_by_columns
         Array(unique_by&.columns)
       end
 
-
```

Comment:
All these newlines were intentional for method grouping.

Reply:
I see... will add newlines back.